src/components/Upgrade/UpgradeBanner.js

```diff
@@ -1,12 +1,15 @@
-import React, { useCallback, useState } from 'react'
+import React, { useEffect, useCallback, useState } from 'react'
 import PropTypes from 'prop-types'
 import { Transition, animated } from 'react-spring'
-import { Button, Viewport, springs } from '@aragon/ui'
+import { Button, springs, useViewport } from '@aragon/ui'
 import Banner, { BANNER_HEIGHT } from '../Banner/Banner'
 import UpgradeModal from './UpgradeModal'
 import { banner } from './content'
 
-const UpgradeBanner = React.memo(({ visible, onUpgrade }) => {
+const SHOW_UPGRADE_MODAL_KEY = 'SHOW_UPGRADE_MODAL_FIRST_TIME'
+const OCTOBER_1ST_2019 = new Date('October 1 2019 00:00').getTime()
```

Comment:
Maybe I could use the `timestamp` directly, thoughts?

Reply:
This is probably fine; this is ideally something we remove before the actual date anyway and is just a back up in case we don't.